### ISSUE TITLE
Queries without remote execution information

### ISSUE THREAD

**Issue by miguel76** [dice-group/LSQ#38]
Note: this is a question, not an actual issue
I would like to know why in the generated data there are queries that have no remote execution information, i.e. there are no outgoing triples with predicate lsqv:hasRemoteExec.
In the data from bioportal, while all 158,553 queries have lsqv:hasLocalExec , only 89,664 have lsqv:hasRemoteExec.
If I understand correctly the model, the remote execution is when the query was originally executed.
So, given that the queries are extracted from existing logs, I would expect that there is always a resource representing the remote execution.
Am I wrong?

**Comment by saleem-muhammad** [comment]
Hi, Thank you for the issue. Each query should have at least one remote execution. Technically, you are right. @Aklakan, any idea why some queries do not have remote execution?

**Comment by Aklakan** [comment]
Without having looked at the specific data yet I can think of two reasons why there could be missing remoteExecs:

* Mismatch of the hashes derived from the queries due to a change of the ID algorithm. At some point we switched from plain md5 strings to sha256 with base64-url-encoding (results in a bit shorter IDs).
* Parse errors in the query; a non-parseable query cannot be executed. One likely reason could be due to use of undeclared prefixes. At some point we added a complete prefix.cc dump to LSQ's default processing because otherwise too many queries were not parseable.
